### DISCUSSION TITLE
Steam TOS

### DISCUSSION THREAD

**Discussion by NinjaZ-553** [SteamRE/DepotDownloader#256] [Q&A]
Hey!
Do you know if it is against steam TOS to download old versions of a game? I really want to use this, but am afraid I will lose my account. I have searched everywhere and can't find an answer.

**Comment by philou-felin** [comment]
I downloaded many times and I still have my account. (Sorry, I know this is not a proper answer.)

**Reply by NinjaZ-553** [reply]
> I downloaded many times and I still have my account. (Sorry, I know this is not a proper answer.)

Thanks
